tools/packgen/test/data/CMakeTestProject/invalid_manifest2.yml

```diff
@@ -0,0 +1,65 @@
+### CMSIS Pack manifest file ###
```

Comment:
Could you add a comment to the yaml file that specifies what makes this file an invalid one?